.github/workflows/trivy-analysis.yml

```diff
@@ -0,0 +1,67 @@
+name: Trivy
+on:
+  workflow_dispatch:
+  # Uncomment this for testing purposes
+  #push:
+  #  branches: [ main ]
+  # Run the job twice a day  
+  schedule:
+    - cron: "0 0,12 * * *"
```

Comment:
Do we have an estimate on when/how often the security database are updated on Trivy side?

Reply:
Trivy downloads its vulnerability database every 12 hours, according to [their documentation](https://aquasecurity.github.io/trivy/v0.17.0/examples/db/#:~:text=Vulnerability%20DB-,Skip%20update%20of%20vulnerability%20DB,the%20%2D%2Dskip%2Dupdate%20option.).